pkg/defkinds/nodejs/definition.go

```diff
@@ -158,11 +151,7 @@ func NewKind(genericDef *builddef.BuildDef) (Definition, error) {
 	}
 
 	if def.BaseImage == "" {
-		baseImage, ok := defaultBaseImages[def.Version]
-		if !ok {
-			return def, xerrors.Errorf("no default base image defined for NodeJS v%s, you have to define it by yourself in your zbuildfile or use one of the supported versions: %s", def.Version, supportedVersions)
-		}
-		def.BaseImage = baseImage
+		def.BaseImage = fmt.Sprintf("docker.io/library/node:%s-buster-slim", def.Version)
```

Comment:
a non existing base image will make a pull error and stop the build ?

Reply:
When calling `zbuild update` with an invalid version/base image, it fails to resolve the image digest. For now there's nothing in place to check if the base image/version has changed but it'll come soon. Thus, yeah in some way, bundle fails/should fail whenever the image is invalid :wink: